src/redux/categories/categories.js

```diff
@@ -0,0 +1,40 @@
+// Initial category
+const initialCategoryState = [];
+
+// create action contants - This are calleds action types
+const ADD_CATEGORY = 'category/ADD_CATEGORY';
+const STATUS_CHECKED = 'category/STATUS_CHECKED';
+
+// Action Creators for the action constant
+
+// ########## Action to category ###############/
+const addCategory = (category) => ({
+  type: ADD_CATEGORY,
+  payload: {
+    title: category.title,
+  },
+});
+
+const statusCheck = () => ({
+  type: STATUS_CHECKED,
+  payload: {
+    title: 'Under Construction',
+  },
+});
+
+// Reducers that recieves the actions
+const categoryReducer = (state = initialCategoryState, action) => {
+  switch (action.type) {
+    case ADD_CATEGORY:
+      return [...state, action.payload];
+    case STATUS_CHECKED:
+      return [...state, action.payload];
+    default:
+      return state;
+  }
```

Comment:
- [ ] Kindly notice that :
* in case `action.type` is `CHECK_STATUS` the function returns a string "Under construction".
* if the `action.type` is different, it returns the state with no changes.
* also there is no need for the `ADD_CATEGORY` type.